packages/core/src/tracing/spanBuffer.js

```diff
@@ -130,6 +137,9 @@ exports.activate = function activate(extraConfig) {
   if (extraConfig && extraConfig.tracing && extraConfig.tracing.spanBatchingEnabled) {
     batchingEnabled = true;
   }
+  if (extraConfig?.tracing?.ignoreEndpoints) {
```

Comment:
suggestion: How about we merge both extraConfig snippets together?
Lets always try to give it a little bit of love.

```
if (extraConfig?.tracing?) {
   spanBatchingEnabled
   ignoreEndpoints
}
```